compiler/ballerina-lang/src/main/java/org/ballerinalang/compiler/plugins/CompilerPlugin.java

```diff
@@ -137,9 +136,9 @@ default void process(ClassDefinition classDefinition, List<AnnotationAttachmentN
      * Notifies when the code generated phase is completed.
      *
      * @param project Project instance
-     * @param target Target directory
+     * @param executablePath path of the executable jar
      */
-    default void codeGenerated(Project project, Target target) {
+    default void codeGenerated(Project project, Path executablePath) {
```

Comment:
This is an API change. We need to communicate this change especially with the C2C team.